src/process_mcp_repos.py

```diff
@@ -458,7 +458,7 @@ def main():
         if failed_forks:
             summary_lines.append("\nFailed Repository Details:")
```

Comment:
Remove extra \n to match normal setup
```suggestion
            summary_lines.append("Failed Repository Details:")
```